src/test/fuzz/txorphan.cpp

```diff
@@ -228,3 +228,154 @@ FUZZ_TARGET(txorphan, .init = initialize_orphanage)
     }
     orphanage->SanityCheck();
 }
+
+FUZZ_TARGET(txorphan_protected, .init = initialize_orphanage)
```

Comment:
Had a thought: this harness could be relaxed to where the fuzzer input selects the protected peers, which could be any subset of all the peers. This would allow coverage of a scenario where no peers exceed reservation limits, and possibly stronger assertions in that case.